interface/patient_file/encounter/forms.php

```diff
@@ -892,42 +878,39 @@ function findPosY(obj) {
         // Create the ESign instance for this form
         $esign = $esignApi->createFormESign($iter['id'], $formdir, $encounter);
 
-        // echo "<tr>"; // Removed as bug fix.
-
-        echo "<td class='border-bottom border-dark w-100 d-flex justify-content-between'>";
-
         // Figure out the correct author (encounter authors are the '$providerNameRes', while other
         // form authors are the '$user['fname'] . "  " . $user['lname']').
-        if ($formdir == 'newpatient') {
-            $form_author = $providerNameRes;
-        } else {
-            $form_author = ($user['fname'] ?? '') . "  " . ($user['lname'] ?? '');
-        }
+        $form_author = ($formdir == 'newpatient') ? $providerNameRes : ($user['fname'] ?? '') . "  " . ($user['lname'] ?? '');
         $div_nums_attr = attr($divnos);
         $title = xla("Expand/Collapse this form");
         $display = text($form_name) . " " . xlt("by") . " " . text($form_author);
         $author_text = text($form_author);
         $by_text = xlt("by");
+        $chevron = ($divnos == 1) ? "fa-chevron-down" : "fa-chevron-down";
```

Comment:
don't know which way is up or which way is down :)

Reply:
Yeah, it seems all the forms start in an open state which means the chevron should always be down. I can probably just drop that conditional statement and start everything with the down arrow.